docs/general/ad-filtering/create-own-filters.md

```diff
@@ -208,14 +208,80 @@ Example:
 
 #### Basic modifiers {#basic-rules-basic-modifiers}
 
+[//]: # (Please, keep them sorted)
```

Comment:
why not just like this?
```
<!-- Please, keep them sorted -->
```

Reply:
Some say that's the most platform-independent :) https://stackoverflow.com/questions/4823468/comments-in-markdown